src/services/edge-firewall-rules-engine-services/v4/edit-edge-firewall-rules-engine-service.js

```diff
@@ -69,12 +69,29 @@ const parseBehaviors = (behaviors) => {
   return parsedBehaviors
 }
 
+const parseCriteria = (criteria) => {
+  const argumentTransformers = {
+    '${network}': (arg) => Number(arg)
+  }
+
```

Comment:
This is pretty much the same code from the previous file (or is there a difference I'm missing)?
Any chance we can move this to a helper (or other layer) to easy up maintenance?